cropper/app/controllers/Application.scala

```diff
@@ -44,52 +50,30 @@ object Application extends Controller with ArgoHelpers {
 
   def index = Authenticated { indexResponse }
 
+  def export = Authenticated.async(parse.json) { httpRequest =>
+    httpRequest.body.validate[ExportRequest] map { exportRequest =>
```

Comment:
`body` is a JValue because of the `parse.json` BodyParser on the line above. Does that answer your question?